### DISCUSSION TITLE
LDM Notes for April 11th, 2022

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#6007] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2022/LDM-2022-04-11.md

## Agenda

1. Relax restrictions on braces on raw interpolated strings
2. Self-type stopgap attribute

**Comment by Joe4evr** [comment]
> Rejected. We will look at a BCL analyzer to enforce the rules they want to enforce for these interfaces.

Okay then, guess I won't need to update my own constraints package *quite* yet.
